echo-webhooks/echo-webhooks.gradle

```diff
@@ -19,6 +19,7 @@ dependencies {
   implementation project(':echo-model')
   implementation "org.springframework.boot:spring-boot-starter-web"
   implementation "io.spinnaker.kork:kork-artifacts"
+  implementation "io.spinnaker.kork:kork-web"
```

Comment:
Shouldn't be necessary.  Kork-core should have this.  The exception class should be coming in transitively automatically via one of the other dependencies vs. the entire web which isn't necessary.

Reply:
Removed this, but currently kork-core doesn't exist in this file.
Adding kork-core dependency to find the exception class as transitive dependency. 
If I don't add any dependency the build fails in finding the exception class.